warehouse/query-core/src/test/java/datawave/query/language/parser/jexl/TestLuceneToJexlQueryParser.java

```diff
@@ -345,10 +345,12 @@ public void testMiscEscape() throws ParseException {
     @Test
     public void testMiscEscapeTokenization() throws ParseException {
         // apostrophe escapes
-        Assert.assertEquals("TOKFIELD == 'johnny\\'s'", parseQuery("TOKFIELD:johnny's"));
-        Assert.assertEquals("TOKFIELD == 'johnny\\'s'", parseQuery("TOKFIELD:johnny\\'s")); // lucene drops single slash
-        Assert.assertEquals("TOKFIELD == 'johnny\\'s'", parseQuery("TOKFIELD:\"johnny's\""));
-        Assert.assertEquals("content:phrase(TOKFIELD, termOffsetMap, 'johnny\\'s', 'chicken')", parseQuery("TOKFIELD:\"johnny's chicken\""));
+        Assert.assertEquals("(TOKFIELD == 'johnny\\'s' || TOKFIELD == 'johnny')", parseQuery("TOKFIELD:johnny's"));
```

Comment:
Fixed an issue with apostrophe handling for tokenized queries. This is the appropriate behavior.